### PR TITLE
src: update cli help for env

### PR DESCRIPTION
**Pull request by matejvasek**
Signed-off-by: Matej Vasek <mvasek@redhat.com>

### CODE REVIEW

**Review by zroubalik — COMMENTED** [review]
Looking good, could you please add a note to docs/guides/commands.md ?

**Comment by matejvasek** [comment]
@zroubalik PTAL

File: docs/guides/func_yaml.md [hunk 1]
```diff
@@ -33,12 +33,13 @@ it's typically unnecessary to modify the `builder` field, using values from
 The `env` field allows you to set environment variables that will be
 available to your function at runtime. For example, to set a `MODE` environment
 variable to `debug` when the function is deployed, your `func.yaml` file
-may look like this.
+may look like this. For explicitly unset variables dash `-` suffix is used.
```
Comment (lance) [inline docs/guides/func_yaml.md:36]: Small wording suggestion
```suggestion
may look like this. To unset variables use a dash suffix. For example, to unset `MODE`, use `MODE-`.
```

**Review by lance — APPROVED** [review]
LGTM with one minor wording suggestion

**Pull request merged**